bin/benchmark.bat

```diff
@@ -0,0 +1,11 @@
+rem @echo off
```

Comment:
Did you mean to leave this commented out?

Reply:
No, wasn't meant like that. Removed the comment.